misc/docker/Dockerfile

```diff
@@ -1,23 +1,22 @@
 FROM busybox
 
 RUN set -x \
-    && wget -O- http://nixos.org/releases/nix/nix-1.9/nix-1.9-x86_64-linux.tar.bz2 | \
+    && wget -O- http://nixos.org/releases/nix/nix-1.10/nix-1.10-x86_64-linux.tar.bz2 | \
```

Comment:
No, there is a generic make rule for doing substitutions. See e.g. `misc/systemd/local.mk` and `misc/systemd/nix-daemon.service.in`.